src/node/hooks.h

```diff
@@ -103,7 +103,9 @@ namespace ccf
 
     void call(kv::ConfigurableConsensus* consensus) override
     {
-      for (auto nc : configs)
+      // This hook is always executed after the hook for the nodes table above,
```

Comment:
Is it possible to create an assert/throw if this assumption is broken

Reply:
Sure, that's a good idea.

Reply:
Good call, the name I had chosen actually broke that. Fixed now.